src/tests/functional/plugin/cpu/single_layer_tests/normalize.cpp

```diff
@@ -136,10 +136,10 @@ const std::vector<ov::Shape> inputShapeStatic_2D = {
 
 const std::vector<InputShape> inputShapeDynamic_2D = {
         {{-1, -1},
-        {{2, 3}, {2, 3}, {5, 5}}},
+        {{2, 3}, {2, 3}, {5, 5}, {2, 3}}},
```

Comment:
please also add `fusingFakeQuantizePerChannel` or `fusingFakeQuantizePerTensor` into `fusingParamsSetDynamic ` to check that fusing with FQ works correct

Reply:
good catch, added!